### PR TITLE
Added 3 Pokemon and fixed up image links.

### PR DESCRIPTION
**Pull request by PrayasJ**
Tl;dr replaced links to form a uniform and structured dataset.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,36 +3,36 @@ Implemeted for a git session - 21/09/2019 (Better idea would have been to use in
...
-| 018 |Pidgeot|Flying|![Pidgeot](https://img.pokemondb.net/artwork/large/pidgeot.jpg)|
-| 019 | Rattata | Normal| ![Rattata](https://vignette.wikia.nocookie.net/gabes-pokemon-adventures/images/2/2b/019Rattata_Dream.png/revision/latest?cb=20151206205539)|
+| 003 | Venusaur | Grass | ![Venusaur](https://assets.pokemon.com/assets/cms2/img/pokedex/full/003.png)|
+|004|Charmander|Fire|![Charmander](https://assets.pokemon.com/assets/cms2/img/pokedex/full/004.png)|
+| 005 |Charmeleon|Flame|![Charmeleon](https://assets.pokemon.com/assets/cms2/img/pokedex/full/005.png)|
+| 006 | Charizard | fire/flyinng | ![charizard](https://assets.pokemon.com/assets/cms2/img/pokedex/full/006.png)|
+| 007 | Squirtle | Water | ![Squirtle](https://assets.pokemon.com/assets/cms2/img/pokedex/full/007.png)|
+| 008 | Wartortle |Water|![Wartortle](https://assets.pokemon.com/assets/cms2/img/pokedex/full/008.png)|
+| 009 | Blastoise |Water|![Blastoise](https://assets.pokemon.com/assets/cms2/img/pokedex/full/009.png)|
+| 010 | Caterpie | Grass | ![Caterpie](https://assets.pokemon.com/assets/cms2/img/pokedex/full/0010.png)|
```
Comment (yashlamba) [inline README.md]: This isn't loading

**Comment by yashlamba** [comment]
I don't think it came out the way you wanted

File: README.md [hunk 2]
```diff
@@ -3,36 +3,38 @@ Implemeted for a git session - 21/09/2019 (Better idea would have been to use in
...
+| 001 | Bulbasaur | ![Grass](https://vignette.wikia.nocookie.net/pokemon/images/4/46/Type_Grass.gif) | ![Bulbasaur](https://assets.pokemon.com/assets/cms2/img/pokedex/full/001.png)|
+| 002 | Ivysaur | ![Grass](https://vignette.wikia.nocookie.net/pokemon/images/4/46/Type_Grass.gif) | ![Ivysaur](https://assets.pokemon.com/assets/cms2/img/pokedex/full/002.png)|
+| 003 | Venusaur | ![Grass](https://vignette.wikia.nocookie.net/pokemon/images/4/46/Type_Grass.gif) | ![Venusaur](https://assets.pokemon.com/assets/cms2/img/pokedex/full/003.png)|
+|004|Charmander|![Fire](https://vignette.wikia.nocookie.net/pokemon/images/4/4d/Type_Fire.gif)|![Charmander](https://assets.pokemon.com/assets/cms2/img/pokedex/full/004.png)|
+| 005 |Charmeleon|![Fire](https://vignette.wikia.nocookie.net/pokemon/images/4/4d/Type_Fire.gif)|![Charmeleon](https://assets.pokemon.com/assets/cms2/img/pokedex/full/005.png)|
+| 006 | Charizard | ![Fire](https://vignette.wikia.nocookie.net/pokemon/images/4/4d/Type_Fire.gif)/flyinng | ![charizard](https://assets.pokemon.com/assets/cms2/img/pokedex/full/006.png)|
```
Comment (yashlamba) [inline README.md]: Flyinng

File: README.md [hunk 3]
```diff
@@ -808,5 +809,24 @@ Implemeted for a git session - 21/09/2019 (Better idea would have been to use in
...
+https://vignette.wikia.nocookie.net/pokemon/images/f/f2/Type_Ghost.gif
+https://vignette.wikia.nocookie.net/pokemon/images/4/46/Type_Grass.gif
+https://vignette.wikia.nocookie.net/pokemon/images/1/1d/Type_Ground.gif
+https://vignette.wikia.nocookie.net/pokemon/images/8/84/Type_Ice.gif
+https://vignette.wikia.nocookie.net/pokemon/images/6/61/Type_Normal.gif
+https://vignette.wikia.nocookie.net/pokemon/images/8/82/Type_Poison.gif
+https://vignette.wikia.nocookie.net/pokemon/images/6/65/Type_Psychic.gif
+https://vignette.wikia.nocookie.net/pokemon/images/b/b3/Type_Rock.gif
+https://vignette.wikia.nocookie.net/pokemon/images/a/ab/Type_Steel.gif
+https://vignette.wikia.nocookie.net/pokemon/images/e/ed/Type_Water.gif
```
Comment (yashlamba) [inline README.md]: Can you turn it to a list with the icons only as they look not the links, people can just copy the whole thing

Comment (PrayasJ) [reply README.md]: done

**Comment by yashlamba** [comment]
Cool thanks!

**Pull request merged**